packages/gatsby-theme-apollo-core/package.json

```diff
@@ -13,12 +13,13 @@
   "dependencies": {
     "@apollo/space-kit": "^5.6.0",
     "@emotion/core": "^10.0.7",
-    "@emotion/styled": "^10.0.7",
-    "@svgr/webpack": "^4.2.0",
-    "gatsby-plugin-emotion": "^4.0.3",
-    "gatsby-plugin-less": "^3.0.1",
-    "gatsby-plugin-react-helmet": "^3.0.2",
-    "gatsby-plugin-svgr": "^2.0.1",
+    "@emotion/react": "^11.0.0",
+    "@emotion/styled": "^11.0.0",
+    "@svgr/webpack": "^5.0.0",
+    "gatsby-plugin-emotion": "^6.0.0",
+    "gatsby-plugin-less": "^5.0.0",
+    "gatsby-plugin-react-helmet": "^4.0.0",
+    "gatsby-plugin-svgr": "3.0.0-beta.0",
```

Comment:
There's an open PR to update this plugin to support Gatsby v3. Someone in the comments [published a beta version](https://github.com/zabute/gatsby-plugin-svgr/pull/51#issuecomment-789350978) in the interim until it gets merged.